src/attributes.adoc

```diff
@@ -48,6 +48,8 @@ endif::[]
 :cheri_pte_ext_name:       Zcheri_pte
 // Extension for CHERI capabilities in vector registers
 :cheri_vectorcap_ext_name: Zcheri_vectorcap
+// Extension for CHERI thread identification
+:cheri_tid_ext_name: Zcheri_tid
```

Comment:
Thanks for factoring this out into another extension. However, I thought that we were hoping this wouldn't be a CHERI-specific instruction, so perhaps it doesn't need the `cheri_` prefix? Also, note that `_` is an illegal character in extension names, so could you remove that too? (We are supposed to fix this in the other extensions but haven't got around to it yet!)

Reply:
Sure, will fix. It is a CHERI-dependent extension because we are introducing capabilities. `utidc` has CLEN.